Tasks/ArchiveFiles/archivefiles.ts

```diff
@@ -128,12 +128,7 @@ function tarArchive(archive: string, compression: string, files: string[]) {
 function handleExecResult(execResult, archive) {
     if (execResult.code != tl.TaskResult.Succeeded) {
         tl.debug('execResult: ' + JSON.stringify(execResult));
-        var message = 'Archive creation failed for archive file: ' + archive +
-            '\ncode: ' + execResult.code +
-            '\nstdout: ' + execResult.stdout +
-            '\nstderr: ' + execResult.stderr +
-            '\nerror: ' + execResult.error;
-        failTask(message);
+        failTask(tl.loc('HandleExec', archive, execResult.code, execResult.stdout, execResult.stderr, execResult.error));
```

Comment:
"HandleExec" is the function, but doesn't convey the meaning of this message.  I suggest a key name such as "ArchiveCreationFailedWithError".